include/z64.h

```diff
@@ -1019,6 +1019,80 @@ typedef struct {
     /* 0x4C */ u32 unk_4C;
 } PreRender; // size = 0x50
 
+#define TRANS_TRIGGER_END -20 // transition is ending (arriving in a new area)
+#define TRANS_TRIGGER_OFF 0 // transition is not active
+#define TRANS_TRIGGER_START 20 // start transition (exiting an area)
```

Comment:
Very minor but I feel like this order would make more sense, at least from a functional standpoint
```suggestion
#define TRANS_TRIGGER_OFF 0 // transition is not active
#define TRANS_TRIGGER_START 20 // start transition (exiting an area)
#define TRANS_TRIGGER_END -20 // transition is ending (arriving in a new area)
```

Reply:
Looking at it with the play state lifecycle, end/off/start is actually the right order though

`Gameplay_Init`: `globalCtx->transitionTrigger = TRANS_TRIGGER_END;`
this leads to `transitionMode = TRANS_MODE_SETUP;`,
eventually `Gameplay_Update` does `globalCtx->transitionTrigger = TRANS_TRIGGER_OFF;`
and the transition fades out,

(play state runs...)

something sets `globalCtx->transitionTrigger = TRANS_TRIGGER_START;`,
leading to `transitionMode = TRANS_MODE_SETUP;`,
eventually the transition fades in
and then `SET_NEXT_GAMESTATE` puts an end to the current play state

Reply:
To be clear I think the end/off/start order should be prefered because that's always how the play state rolls (afaik, besides the debug combo for switching to map select)

off/start/end is the right order too but only if switching from play to play, and there are important cases where play switches to/from another state

Reply:
i had it in that order originally just cause it was the order of the values from largest to highest. but yea i can see an argument for both

from a user standpoint youll be comparing against OFF sometimes and triggering it to start when you use it, so I can see why romans order makes sense from the users standpoint, as opposed to the order of things in the perspective of the play gamestate

I dont think i have a strong preference one way or another here but Ill put some more thought into it

Reply:
I went with changing the order just cause I think it does makes sense to see the different types listed in that way. I still agree that the other order makes sense in the context of the play life cycle, but guess I see that as less important over having it be in a more natural order